File: JS/URI/1046.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");

var a = parseInt(line[0]);
var b = parseInt(line[1]);

if(a == b)
{
    console.log("O JOGO DUROU 24 HORA(S)");
}

else if(a > b)
{
    let total = a - 24;
    total = Math.abs(total) + b;
    console.log("O JOGO DUROU "+total+" HORA(S)");

}

else
{
    total = b - a;
    console.log("O JOGO DUROU "+total+" HORA(S)");

}

File: C++/1046.cpp
#include<iostream>

using namespace std;

int main(){


    int a, b, total;

    cin >> a >> b;

    
        if (a == b)
            cout << "O JOGO DUROU 24 HORA(S)" << "\n";
        else if (a > b)
        {
            total = a - 24;
            total = abs(total) + b;
            cout << "O JOGO DUROU " << total << " HORA(S)\n";
        }
        else
        {
            total = b - a;
            cout << "O JOGO DUROU " <<  total << " HORA(S)\n";
        }

}